android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -74,11 +76,12 @@
 
     private String searchText = "";
     private SearchView searchView;
+    final ArrayList<String> dateList = new ArrayList<>();
 
-    private DayScheduleAdapter bookMarksListAdapter;
+    private GlobalSearchAdapter bookMarksListAdapter;
```

Comment:
Global Search is meant for the user only. GlobalSearchAdapter can be renamed into GenericAdapter

_Sent from my OnePlus ONEPLUS A5000 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_